stdpopsim/genetic_maps.py

```diff
@@ -53,8 +50,25 @@ def __init__(
 
     @property
     def map_cache_dir(self):
+        """
+        The path to the directory in which the files for this map are stored.
+        Names for individual chromosome files within this directory can be obtained
+        using :attr:`file_pattern`.
+
+        :type: pathlib.PosixPath
+        """
         return self._cache.cache_path
 
+    @property
+    def file_pattern(self):
+        """
+        The pattern used to map name individual chromosome to files, suitable for use
```

Comment:
This sounds a bit clunky. Maybe just "the pattern used to map indiv..."? Also, as long as you're documenting this, would you also be able to mention the `id` variable? I'm not sure of the correct nomenclature here, but maybe @jeromekelleher knows?
```python
>>> gmap.file_pattern
'genetic_map_GRCh37_{id}.txt'
>>> gmap.file_pattern.format("40")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 'id'
>>> gmap.file_pattern.format(id="40")
'genetic_map_GRCh37_40.txt'
```

Reply:
Sure. I just copied it from the original docs in the class header without thinking too hard!

Reply:
And you are right - the grammar's all wrong. I hadn't even read it properly!